drivers/hmis/app/graphql/types/hmis_schema/enums/ce_match_rule_type.rb

```diff
@@ -1,16 +0,0 @@
-###
```

Comment:
Removing this since it's unused. We now resolve the 2 different types separately on the Opportunity: https://github.com/greenriver/hmis-warehouse/pull/5013/files#diff-4191bd5a0be0f9137f0e67c2aeb2702854afd3ba99d43bb5179ba418f11d8276R20-R21